src/wallet/api.rs

```diff
@@ -785,6 +779,12 @@ impl Wallet {
         Ok(all_utxos)
     }
 
+    pub fn get_all_locked_utxo(&self) -> Result<Vec<ListUnspentResultEntry>, WalletError> {
+        let all_utxos = self
+            .rpc
+            .list_unspent(Some(0), Some(9999999), None, None, None)?;
+        Ok(all_utxos)
+    }
```

Comment:
Whats the difference between this and `get_all_utxo`? We are filtering out fidelity and swap coins manually. Do we need to have a separate API? 

I have a feeling we can remove all these locks/unlocks from the API to simplify the logic. We should always unlock everything and then manually filter out stuffs that we don't want to spend from. Can be done in a follow up PR.

Reply:
I added `get_all_locked_utxo` because `get_all_utxo` was unlocking all the UTXOs, including the locked ones. During the coin selection process, it was fetching even the locked UTXOs as well.

Reply:
My read is we needed this locking/unlocking shenanigan because we were relying on core for transaction creation. Which isn't the case anymore. 

Can we just remove all locking/unlocking stuffs and see if anything breaks? If it does I am sure we can fix them by correct filters at the call site. But better to do it in separate PR.